README.md

```diff
@@ -45,6 +45,8 @@ and instance of the simplepicker.
     - `` (`boolean`): If `true` disables the time picker section.
     - `compactMode` (`boolean`): If `true` the simplepicker will be more compact; the large
                                  display of the selected date, i.e. 25th will not be displayed.
+    - `selectedDate` (`Date`): initialize the simplepicker with this date, if not used then today
+                               will be used
```

Comment:
What the story behind documenting this?

Reply:
Ah, looking that the changes below, you call `this.reset(this.selectedDate)` but this does not work practically. Let me explain:
 - The `selectedDate` is set to today's date in the constructor which and the `this.reset(this.selectedDate)` is called in the init method which is called by the constructor. _So_ updating this does not do anything so we don't need to document this.

Does the explanation make sense?

Reply:
Also, you will notice that now is not using the this.selectedDate, but the this.opts.selectedDate the reset now checks if the date is "undefined" if so then it will use the today so it works in the same way it worked before, I'm using it at the moment and I dont see any side effect as is.